cmd/skopeo/tags.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/containers/image/v5/docker"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+
+	"io"
+)
+
+// tagListOutput is the output format of (skopeo tags), primarily so that we can format it with a simple json.MarshalIndent.
+type tagListOutput struct {
+	DockerRepository string   `json:",omitempty"`
```

Comment:
I prefer calling it just "Repository". The distribution work in the OCI is ongoing which will make the protocol truly independent. Same in the man page.